rnd/autogpt_builder/.env.example

```diff
@@ -1,4 +1,5 @@
-AGPT_SERVER_URL=http://localhost:8000/api
+NEXT_PUBLIC_AGPT_SERVER_URL=http://localhost:8000/api
+NEXT_PUBLIC_AGPT_MARKETPLACE_URL=http://localhost:8001/market
```

Comment:
Dont these need to be updating to be ``/api/v1/market``

Reply:
Yes, done